model/src/main/java/org/mskcc/cmo/metadb/model/SampleManifestEntity.java

```diff
@@ -135,6 +136,14 @@ public UUID getUuid() {
     public void setUuid(UUID uuid) {
         this.uuid = uuid;
     }
+    
+    public Boolean getIsTumor() {
```

Comment:
Should this just be (tumorOrNormal.equals("TUMOR") ?  I don't suggest we hardcode the string - we should use an Enum and make sure we know the vocabulary for this field.